features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
+Items can be bound to bindings i.e. for reading the status from e.g. Sonos, or for updating them. Read the wiki page for the respective binding for more help and examples.
```

Comment:
not the wiki page, the docs page please :) You should also cross reference as often as possible. So "docs page on bindings" should be a link there